demos/link-prediction/random-walks/cora-lp-demo.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>How long does <code>evaluate_link_prediction_model</code> take to run? If it's relatively short, potentially we don't need to have the <code>Finished training link prediction model</code> line, which makes the output below slightly harder to parse?</p><p> </p><p>In addition. In fact, we could potentially remove all of the <code>print</code> s here and instead display the results via a Pandas DataFrame, since that has a nice Jupyter rendering and makes things aligned, e.g.:</p><p> </p><pre class="ql-syntax" spellcheck="false">def run_link_prediction(...):
    clf = ...
    score = ...
    return {"classifier": ..., ...}
</pre><p> </p><pre class="ql-syntax" spellcheck="false">results = [run_link_prediction(op) for op in ...]
best_results = ...

print(f"Best result ...")

pd.DataFrame(
    [(result["binary_operator"].__name__, result["score"] for result in results],
    columns=("name", "ROC AUC score")
)
</pre><p>(This does flip the "Best result" printing to be before the dataframe, but I think that's ok.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Nice :+1:</p>